<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Untitled - LitePad v0.01</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/litepad-96x96.png</normaloff>:/resources/litepad-96x96.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background : #efefef; color : #000000; selection-background-color : #0055ef; selection-color: #ffffff;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">background : #ffffff; color : #000000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Find">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionFindNext"/>
    <addaction name="actionFindPrevious"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete_All_Text"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionFont"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menu_Find"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-find.png</normaloff>:/resources/litepad-find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="statusTip">
    <string>Search for a text string.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-about.png</normaloff>:/resources/litepad-about.png</iconset>
   </property>
   <property name="text">
    <string>About LitePad</string>
   </property>
   <property name="statusTip">
    <string>Get information about the application.</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
   <property name="statusTip">
    <string>Get information about Qt, the program used to code this program.</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-find-replace.png</normaloff>:/resources/litepad-find-replace.png</iconset>
   </property>
   <property name="text">
    <string>Replace...</string>
   </property>
   <property name="statusTip">
    <string>Replace a text string.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-find.png</normaloff>:/resources/litepad-find.png</iconset>
   </property>
   <property name="text">
    <string>Find Next</string>
   </property>
   <property name="statusTip">
    <string>Find the next occurrence.</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionFindPrevious">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-find.png</normaloff>:/resources/litepad-find.png</iconset>
   </property>
   <property name="text">
    <string>Find Previous</string>
   </property>
   <property name="statusTip">
    <string>Find the previous occurrence.</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad_new.png</normaloff>:/resources/litepad_new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Create a new text document and discard the current one.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad_open.png</normaloff>:/resources/litepad_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open a new text document.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad_save.png</normaloff>:/resources/litepad_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save the current text document.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad_save.png</normaloff>:/resources/litepad_save.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="statusTip">
    <string>Save the current text document.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-exit.png</normaloff>:/resources/litepad-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Quit the application.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-undo.png</normaloff>:/resources/litepad-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="statusTip">
    <string>Undo the previous action.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-redo.png</normaloff>:/resources/litepad-redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="statusTip">
    <string>Redo the previous action.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-cut.png</normaloff>:/resources/litepad-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="statusTip">
    <string>Cut the current selection of text.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-copy.png</normaloff>:/resources/litepad-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy the current selection of text.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-paste.png</normaloff>:/resources/litepad-paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste the contents of your clipboard.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete_All_Text">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-delete.png</normaloff>:/resources/litepad-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete All Text</string>
   </property>
   <property name="statusTip">
    <string>Erase all of the text in the current document.</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-paste.png</normaloff>:/resources/litepad-paste.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="statusTip">
    <string>Select all the text in the current document.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/litepad-font.png</normaloff>:/resources/litepad-font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="statusTip">
    <string>Change the current font of the selected text.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
